src/test/java/com/google/devtools/build/lib/sandbox/LinuxSandboxedSpawnRunnerTest.java

```diff
@@ -203,6 +205,22 @@ public void hermeticTmp_sandboxTmpfsOnTmp_tmpNotCreatedOrMounted() throws Except
     assertThat(args).doesNotContain("-m /tmp");
   }
 
+  @Test
+  public void cgroup_canDisableWithExecReq() throws Exception {
```

Comment:
Can we add another test case to verify the original behavior is different and working as intended?

Reply:
ping @AlessandroPatti